src/platform/psmove_port_windows.c

```diff
@@ -788,15 +788,25 @@ psmove_port_set_socket_timeout_ms(int socket, uint32_t timeout_ms)
 void
 psmove_port_sleep_ms(uint32_t duration_ms)
 {
-    HANDLE timer;
+    HANDLE timer = NULL;
```

Comment:
This assignment is unnecessary, because the variable is set below.

Feel free to move the initialisation directly to the declaration:

    HANDLE timer = CreateWaitableTimer(NULL, NULL, NULL);